pkg/iostreams/iostreams.go

```diff
@@ -52,6 +54,32 @@ func (s *IOStreams) ColorSupport256() bool {
 	return s.is256enabled
 }
 
+func (s *IOStreams) ResolveBgColor() string {
+	style := os.Getenv("GLAMOUR_STYLE")
+	if (!s.ColorEnabled()) ||
+		(style != "" && style != "auto") ||
```

Comment:
Yeah I agree. This was my biggest issue with this approach. If` GLAMOUR_STYLE` is set we don't want to actually detect the terminal background color because we are not going to use those results, but right now we are throwing away what the GLAMOUR_STYLE is set to. I thought about setting `TerminalBackgroundColor` to be `GLAMOUR_STYLE` which would save use from having to do it in the `markdown` package but then the naming felt wrong since it is not actually the `TerminalBackgroundColor`. Which is how I ended up doing the check twice. I am happy to move in the direction I outlined above, or if there are better ideas I am open to hearing them.